account-and-billing/account-and-billing.helper.ts

```diff
@@ -0,0 +1,172 @@
+import { CoreConstants } from '../../../components/html/core-constants';
+import { WaitHelper } from '../../../components/html/wait-helper';
+import { StepLogger } from '../../../../core/logger/step-logger';
+import { AccountAndBillingPage } from './account-and-billing.po';
+import { AccountAndBillingConstant } from './account-and-billing.constant';
+import { ExpectationHelper } from '../../../components/misc-utils/expectation-helper';
+import { CommonPageHelper } from '../common/common-page.helper';
+import { PageHelper } from '../../../components/html/page-helper';
+
+const { text: textConst } = AccountAndBillingConstant.attributes;
+const labelsPO = AccountAndBillingPage.labels;
+const linksPO = AccountAndBillingPage.links;
+
+export class AccountAndBillingHelper {
+    static async verifyPage() {
+        await CommonPageHelper.switchToPageFrame();
+        StepLogger.subVerification('Account and Billing page is displayed');
+        await ExpectationHelper.verifyDisplayedStatus(
+            labelsPO.accountAndBillingPageHeader,
+            textConst.accountAndBilling);
+    }
+
+    static async verifyPastPaymentsHeader() {
+        StepLogger.subVerification('Past Payments header is displayed');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400120)
```suggestion
        StepLogger.subVerification("Past Payments header is displayed");
```